src/index.css

```diff
@@ -35,29 +57,136 @@ ul {
 
 .books-list {
   display: flex;
-  width: 250px;
-  justify-content: space-between;
+  height: 7rem;
+  width: 70%;
+  background-color: #fff;
+  padding: 5px 1.688rem;
+  margin-left: 4.25rem;
+}
+
+.left-block {
+  display: flex;
+  width: 30rem;
+  flex-direction: column;
+  height: 20rem;
+}
+
+.right-block {
+  display: flex;
+  flex-direction: column;
+  padding: 10px 10% 1rem;
 }
 
 #booklist-display {
   display: flex;
+  flex-direction: column;
   gap: 5px;
 }
 
 a {
   text-decoration: none;
-  color: blue;
-  font-size: 20px;
+  color: #121212;
+  opacity: 50%;
+  font-size: 17px;
 }
 
-form {
+#form {
   display: flex;
   flex-direction: column;
   gap: 10px;
+  padding-left: 5.25rem;
+  margin-top: 20px;
+}
+
+form {
+  display: flex;
+  gap: 10px;
+}
+
+input {
+  width: 19.5rem;
+  height: 2.813rem;
+  border-radius: 4px;
+  border: solid 1px #e8e8e8;
+  background-color: #fff;
+  padding: 10px;
 }
 
-input,
 .input-submit {
-  height: 2rem;
-  width: 150px;
+  color: #fff;
+  width: 10.5rem;
+  height: 2.713rem;
+  border-radius: 3px;
+  border: none;
+  background-color: #0290ff;
+}
+
+.author {
+  color: #4386bf;
+}
+
```

Comment:
- [ ] The paragraph tag with the `class="category"` is black on your design while it looks grey on the given design. In addition, It is bold and font-size is relatively small compared to how it looks currently. Please address all the mentioned differences. Take a look at the comparison below 👇 
**Your design**
![image](https://user-images.githubusercontent.com/90222110/201371993-ce61f6d2-ec6c-4d68-b736-0a9d47e3ad0f.png)

**Given design**
![image](https://user-images.githubusercontent.com/90222110/201372131-65223f4a-4ef1-4456-96cd-d979894e890b.png)

Reply:
This has been resolved